common/compat.h

```diff
@@ -381,6 +381,29 @@ int        isatty           (int fd);
 
 #endif /* HAVE_ISATTY */
 
+#ifdef HAVE_READPASSPHRASE
+
+#ifdef HAVE_READPASSPHRASE_H
```

Comment:
just a question: so there might be a situation where `HAVE_READPASSPHRASE` is defined but no header `HAVE_READPASSPHRASE_H`? Should we error in such case?

Reply:
Good point; I just removed the header check so it should fail at compile time.